README.rst

```diff
@@ -12,12 +12,23 @@ Installation
     pip install polpo@git+https://github.com/geometric-intelligence/polpo.git@main
 
 
-Or, to also install all the dependencies:
-
+and to also install all the dependencies,
 
 ::
 
     pip install polpo[all]@git+https://github.com/geometric-intelligence/polpo@main
 
-Or the classic pipeline ``clone + pip install .``.
+
+Alternatively, you can use the classic pipeline: 
+::
+    git clone https://github.com/geometric-intelligence/polpo.git
+    
+::
+
+and then create an environment with all required dependences:
```

Comment:
Maybe we should separate this.
Meaning: we keep as before, and then we add an extra paragraph suggesting these things can be done within a virtual environment (and show how it can be created).

This because there's no difference between this command and the above ones. Users can do the installation within venvs or not, so we just suggest a possibility, but without mixing concepts.